### PR TITLE
@eessex => Artwork credits + restyle artwork captions

### PR DESCRIPTION
**Pull request by kanaabe**
Adds support for credits in artworks and restyles captions a bit. Addresses https://github.com/artsy/publishing/issues/306

image
image

### CODE REVIEW

File: src/Components/Publishing/Sections/ArtworkCaption.tsx [hunk 1]
```diff
@@ -189,27 +212,34 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
...
-              {this.renderArtists()}
-            </span>
+          <ArtistName>
+            {this.renderArtists()}
+          </ArtistName>
 
-            {this.renderTitleDatePartner()}
-          </Truncator>
+          <div>
+            <Truncator>
```
Comment (eessex) [inline src/Components/Publishing/Sections/ArtworkCaption.tsx]: Seems like Truncator could be broken out into its own component.

Comment (kanaabe) [reply src/Components/Publishing/Sections/ArtworkCaption.tsx]: Mmm I think it is?

**Comment by kanaabe** [comment]
Realized we don't have any unit tests for `ArtworkCaption` so I'd like to include that in this PR as well --

File: src/Components/Publishing/Sections/ArtworkCaption.tsx (hunk 1 above)
Comment (eessex) [reply src/Components/Publishing/Sections/ArtworkCaption.tsx]: Own file rather

File: src/Components/Publishing/Fixtures/Components.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const Artworks = [
     type: "artwork",
     id: "589a6291275b2410d1beb6a5",
     slug: "fernando-botero-nude-on-the-beach",
-    date: "2000",
+    date: "",
     title: "Nude on the Beach",
```
Comment (eessex) [inline src/Components/Publishing/Fixtures/Components.ts]: Curious why you wanted to remove the date here?

Comment (kanaabe) [reply src/Components/Publishing/Fixtures/Components.ts]: The fixture is meant to represent an artwork with missing info

**Pull request merged**